code/modules/surgery/autopsy.dm

```diff
@@ -0,0 +1,49 @@
+/datum/surgery/autopsy
+	name = "Autopsy"
+	steps = list(/datum/surgery_step/incise, /datum/surgery_step/retract_skin, /datum/surgery_step/saw, /datum/surgery_step/clamp_bleeders,
+				 /datum/surgery_step/autopsy, /datum/surgery_step/close)
+	target_mobtypes = list(/mob/living)
+	possible_locs = list(BODY_ZONE_CHEST)
+	requires_tech = FALSE
+
+/datum/surgery/autopsy/can_start(mob/user, mob/living/carbon/target)
+	. = ..()
+	if(target.stat != DEAD)
+		return FALSE
+
+/datum/surgery_step/autopsy
+	name = "analyze cause of death"
+	implements = list(/obj/item/detective_scanner = 100, TOOL_HEMOSTAT = 50, /obj/item/pen = 25)
+	time = 20
+	experience_given = 20
+	var/autopsyPrintCount = 0
+
+/datum/surgery_step/autopsy/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	user.visible_message("<span class='notice'>[user] starts analyzing [target].</span>", "<span class='notice'>You start analyzing  [target].</span>")
+
+/datum/surgery_step/autopsy/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery, default_display_results = FALSE)
+	var/obj/item/paper/N = new(get_turf(target))
+	var frNum = ++autopsyPrintCount
+
+	N.name = text("AR-[] 'Autopsy Record'",frNum)
+	N.info = text("<center><B>Autopsy Record - (FR-[])</B></center><HR><BR>",frNum)
+	N.info += "<HR><B>Notes:</B><BR>"
+
+	for(var/O in target.damage_record)
+
+		if(istype(O,/obj/item))
+			var/obj/item/I = O
+			N.info += "Analysis shows that [target] was attacked by a  [I.sharpness == IS_BLUNT ? "blunt" : "sharp"] object causing [I.autopsy_description != null ? "[I.autopsy_description] of " : " [I.force > 10 ? "[I.force > 20 ? "devastating" : "deep" ]" : "shallow" ] "] [I.damtype] damage.<BR><BR>" //double breaks for easier reading
```

Comment:
If there is an autopsy description the message will be something like "Lacerations of brute damage", which sounds a bit awkward. I'd change it to either just "Lacerations", or "Lacerations causing [severity] [type] damage"

Reply:
hmm that is a good proposition

Reply:
implemented